### PR TITLE
PW Field to User table

### PR DESCRIPTION
**Pull request by BProw**
Hashed pw field to User table

### CODE REVIEW

File: repos/user.go [hunk 1]
```diff
@@ -41,6 +45,7 @@ type User struct {
 	ID        int64     `json:"-"`
 	FullName  string    `pg:",notnull"`
 	Email     string    `pg:",notnull,unique"`
+	PW 		  string 	`pg:",notnull,unique"`
```
Comment (rooftopsparrow) [inline repos/user.go:48]: PW does not need to be unique across rows.

File: web/api.go [hunk 2]
```diff
@@ -89,10 +89,14 @@ func main() {
...
+
 		// Hash user password.
-		encryptPassword(details.Password)
 		// *** TODO: Hash pw added to CreateUser/db ****
-		user, err := repo.Users.CreateUser(details.Name, details.Email)
+		pwEncyrpted := encryptPassword(details.Password)
+
+		user, err := repo.Users.CreateUser(details.Name, 
+										  	details.Email,
+											  pwEncyrpted)
```
Comment (rooftopsparrow) [inline web/api.go:99]: weird formatting?

**Review by rooftopsparrow — CHANGES_REQUESTED** [review]

**Comment by BProw** [comment]
Thanks! Will update w/ changes reflected shortly.

**Review by BProw — COMMENTED** [review]
Should now be complete. Formatting is strange, what tool do you use for formatting go files in vsc?

**Comment by rooftopsparrow** [comment]
Run `go fmt` to fix go files.

**Comment by rooftopsparrow** [comment]
It looks like the tests are failing too. The tests are using the CreateUser call, should be a simple update.

**Comment by BProw** [comment]
> Run `go fmt` to fix go files.

Thanks! This should work.

> It looks like the tests are failing too. The tests are using the CreateUser call, should be a simple update.

Error when trying to test changes to CreateUser call in feature_test.go:

panic: FATAL #28P01 password authentication failed for user "postgres"

I made sure env password is correct. Currently stuck troubleshooting. Any recommendations?

**Comment by rooftopsparrow** [comment]
I assume its environment variable issue. Try pushing your code against the CI using it's environment variables.

**Comment by BProw** [comment]
> I assume its environment variable issue. Try pushing your code against the CI using it's environment variables.

Definitely agree. I think I already have? Not confident. If you're available this week to chat about this, let me know.

**Comment by rooftopsparrow** [comment]
> Definitely agree. I think I already have?

Ok the error I see in CI is:

```
Error: ./feature_test.go:131:8: no new variables on left side of :=
```

Not anything about environment variables

**Pull request merged**